docs/reference/supported-environments.md

```diff
@@ -95,14 +95,14 @@ Camunda Helm chart version `13.x.x` works with Camunda version `8.8.x`. Check th
-| Web Modeler                                                | -            | PostgreSQL 13.x, 14.x, 15.x, 16.x, 17.x<br/>or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x, 16.x<br/>or Oracle 19c, 23ai<br/>or Microsoft SQL Server (MSSQL) 2019, 2022                                                                                                         |
-| Self-Managed Console                                       | -            | -                                                                                                                                                                                                                                                                             |
+| Component                                                  | Java version | Other requirements                                                                                                                                                                                                                                                                                     |
+| ---------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Orchestration Cluster (Zeebe, Operate, Tasklist, Identity) | OpenJDK 21+  | Elasticsearch 8.16+<br/>Amazon OpenSearch 2.17+ (requires use of [OpenSearch exporter](/self-managed/components/orchestration-cluster/zeebe/exporters/opensearch-exporter.md))                                                                                                                         |
+| Optimize                                                   | OpenJDK 21+  | Elasticsearch 8.16+<br/>Amazon OpenSearch 2.17+                                                                                                                                                                                                                                                        |
+| Connectors                                                 | OpenJDK 21+  |                                                                                                                                                                                                                                                                                                        |
+| Management Identity                                        | OpenJDK 17+  | Keycloak 25.x, 26.x<br/>PostgreSQL 14.x, 15.x, 16.x, 17.x, or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x, 16.x, 17.x (required for [certain features](/self-managed/components/management-identity/miscellaneous/configuration-variables.md#database-configuration))<br/>Oracle 19c<br/>SQL Server 2019 |
```

Comment:
I'd suggest calling SQL Server by its official name. The same applies for the similar changes below.

```suggestion
| Management Identity                                        | OpenJDK 17+  | Keycloak 25.x, 26.x<br/>PostgreSQL 14.x, 15.x, 16.x, 17.x, or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x, 16.x, 17.x (required for [certain features](/self-managed/components/management-identity/miscellaneous/configuration-variables.md#database-configuration))<br/>Oracle 19c<br/>Microsoft SQL Server 2019 |
```